examples/autoscaler-webhook/go.mod

```diff
@@ -5,19 +5,30 @@ go 1.20
 require agones.dev/agones v1.35.0
 
 require (
+	github.com/emicklei/go-restful/v3 v3.9.0 // indirect
```

Comment:
Thank you for updating these while in here - my tooling kept automatically updating these 👍🏻